tests/load-tests/ci-scripts/utility_scripts/playwright-update-tokens.py

```diff
@@ -91,28 +103,46 @@ def thread(user):
         user["token"] = input_token_value
         return user
 
+
+def process_it(output_queue, user):
+    try:
+        output_queue.put({"result": workload(user)})
+    except Exception as e:
+        output_queue.put({"exception": e})
+
+
 def main():
 
     with open("users.json", "r") as fd:
         users = json.load(fd)
 
     users_new = []
-    users_allowlist = []   # keep empty to allow all
-    with concurrent.futures.ThreadPoolExecutor(max_workers=1) as executor:
-        futures = {executor.submit(thread, user): user["username"] for user in users if users_allowlist != [] and user["username"] in users_allowlist}
-        for future in concurrent.futures.as_completed(futures):
-            username = futures[future]
-            try:
-                user = future.result()
-            except Exception as exc:
-                print(f"Failed processing {username}: {exc}")
+    users_allowlist = []  # keep empty to allow all
+
+    for user in users:
+        if users_allowlist is not [] and user["username"] not in users_allowlist:
+            continue
+        result_queue = multiprocessing.Queue()
+        process = multiprocessing.Process(target=process_it, args=(result_queue, user))
+        process.start()
+        try:
+            output = result_queue.get(timeout=100)
+        except queue.Empty:
+            process.terminate()
+            print(f"Timeout processing {user['username']}")
+        else:
+            if "result" in output:
+                users_new.append(output["result"])
+                print(f"Completed processing {user['username']}")
+            elif "exception" in output:
+                print(f"Failed processing {user['username']}: {output['exception']}")
             else:
-                print(f"Finished processing {username}")
-                users_new.append(user)
+                print("Some crazy error")
```

Comment:
could be you forgot this as a debug message ?

Reply:
IMO if should never land in this branch, but I wanted to make it print something scary enough :)